documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Process_Layer/3_4_4_Exchanging_Data.md

```diff
@@ -63,6 +63,32 @@ to switch from data pulling to data pushing easily. The sequence is depicted in
 
 ##### Figure 3.4.4.2: Out-of-band Data Exchange
 
+#### Semantic Interoperability
+
+Semantic Interoperability is crucial for Data Exchange in Data Spaces. This starts dhring
+Design-time, as described in the [Data Offering process](./3_4_2_Data_Offering.md#data-offering),
+and in the process of [publishing](./3_4_2_Data_Offering.md#data-provider-registering-self-descriptions)
+and [quering](./3_4_2_Data_Offering.md#data-consumer-searching-for-self-descriptions) self-descriptions.
+In the process of Data Exchange the Vocabulary Hub and the Vocabularies are invoked for semantic
+interoperability.
+
+The following steps are followed:
+
+1. Prior to the invokation of a data operation (in a synchronous or asynchronous way) by the data consumer one or multiple vocabularies
+are loaded by the data consumers connector
+2. Based on the vocabularies the data consumer can implement required interfaces to conduct the
+data transfer as described in the sections above.
+3. During the data transfer, respectivly after the data was transfered, the data can be validated
+against the given vocabularies. In case of invalid data the consumers connector may decline the data
+usage.
+4. Subsequently to the data transfer the data might require some additonal treatment to make use of
+this, like ETL (Extract, Transform, Loading) tooling. During this activity, a connector can make
+use of Data Apps, as described in the following section.
+
+![Semantic Interoperability](media/semantic_interop_data-transfer.drawio.png)
```

Comment:
Link the steps here to the text in section 2 and the results from the IEC paper to foster the reasoning for it.

Reply:
fixed